docs/docker-image-details.md

```diff
@@ -60,6 +64,10 @@ docker run --rm -d -p 9001:9001 apache/gravitino-iceberg-rest:0.7.0-incubating
 
 Changelog
 
+
+- apache/gravitino-iceberg-rest:0.8.0-incubating
```

Comment:
The latest main branch contains the following statement,  seems missing?

```
- apache/gravitino-iceberg-rest:0.8.0-incubating
  - Supports OSS and ADLS storage.
```

Reply:
could you add the following content?
```
  - Supports event listener.
  - Supports audit log.
```

Reply:
Done.